proto/nym/vpn.proto

```diff
@@ -129,15 +102,7 @@ service NymVpnd {
   rpc VpnDisconnect (DisconnectRequest) returns (DisconnectResponse) {}
   rpc VpnStatus (StatusRequest) returns (StatusResponse) {}
   rpc ImportUserCredential (ImportUserCredentialRequest) returns (ImportUserCredentialResponse) {}
-  // rpc ListenToConnectionStatus (Empty) returns (stream ConnectionStatusUpdate) {}
-  // // Cancel any connection pending state (connecting, disconnecting etc)
-  // // and return to disconnected state
-  // rpc Cancel (Empty) returns (Empty) {}
-  // rpc GetEntryNodeLocations (Empty) returns (LocationListResponse) {}
-  // rpc GetExitNodeLocations (Empty) returns (LocationListResponse) {}
-  // rpc GetFastestEntryGateway (Empty) returns (GatewayResponse) {}
-  // rpc GetFastestExitGateway (Empty) returns (GatewayResponse) {}
-  // rpc SetVpnMode (SetVpnModeRequest) returns (Empty) {}
-  // rpc GetVpnMode (Empty) returns (GetVpnModeResponse) {}
+  rpc ListenToConnectionStateChanges (Empty) returns (stream ConnectionStateChange) {}
+  rpc ListenToConnectionStatus (Empty) returns (stream ConnectionStatusUpdate) {}
```

Comment:
I'd suggest to use shorter names for these new rpc methods, like using `ConnectionState` and `ConnectionStatus` as it's obvious reading the method's signatures and return types it's rpc streaming (and so avoiding redundancy)

as ref this is the official proto def of grpc tutorial:
https://github.com/grpc/grpc-go/blob/master/examples/route_guide/routeguide/route_guide.proto

Reply:
A distinction that is worth taking into account here is that unlike e.g `ListFeatures` in that example these streams are indefinite, while `ListFeatures` as far as I can tell will stream a set of entries from an existing container. This would not be apparent from the method's signature and I think is significant enough to include in the name